demisto_sdk/commands/validate/validators/PB_validators/PB103_is_playbook_has_unconnected_tasks.py

```diff
@@ -0,0 +1,60 @@
+from __future__ import annotations
+
+from typing import Iterable, List
+
+from demisto_sdk.commands.content_graph.objects.playbook import Playbook
+from demisto_sdk.commands.validate.validators.base_validator import (
+    BaseValidator,
+    ValidationResult,
+)
+
+ContentTypes = Playbook
+
+
+class IsPlaybookHasUnconnectedTasks(BaseValidator[ContentTypes]):
+    error_code = "PB103"
+    description = "Validate whether there is an unconnected task."
+    rationale = "Validate whether there is an unconnected task to the root task."
```

Comment:
```suggestion
    rationale = "Make sure there're no unconnected tasks to ensure the playbook will work as expected."
```
rationale should be something like this.